### PR TITLE
feat-6

### LINKED ISSUE #6: FEAT: Create Alunos API on Front-End
### Description
This issue implies creating the axios's alunos API in the services folder and the respective Pinia store in the stores folder.

### Checklist
- [x] Create an `alunosService.js` file in the `services` folder.
- [x] Implement API calls for Alunos (e.g., getAlunos, createAluno, updateAluno, deleteAluno).
- [x] Create an `alunosStore.js` file in the `stores` folder.
- [x] Implement state management and actions for Alunos using Pinia.
- [x] Test the API and store by fetching and displaying Alunos data in a component.

### Notes
- Ensure the API and store are correctly integrated and functional.
- Follow best practices for service and store implementation.
- Please check off each item on the Checklist as you complete the tasks.

### PR DESCRIPTION
**Pull request by Joaovictor23t4**
#### Actions Taken:
- Created API and state management for Alunos
- Fix baseURL of all Cursos and Turmas calls
- Tested each Alunos API call

#### Required Tests:
1. Test each Alunos API call and verify that data is returned correctly

#### Tested Browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari
- [ ] Opera

#### Closing Keywords:
- closes #6

#### Notes:

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <script setup>
```
Comment (Hiansdt) [inline src/views/HomeView.vue:1]: Remover linhas comentadas

**Review by Hiansdt — CHANGES_REQUESTED** [review]
Remove unused logs.

Tip: Use the native search function of VSCode to search for specific lines like logs throughout the whole project.

File: src/views/HomeView.vue (hunk 1 above)
Comment (Joaovictor23t4) [reply src/views/HomeView.vue:1]: Removed!

**Review by Hiansdt — CHANGES_REQUESTED** [review]
OK

**Review by Hiansdt — APPROVED** [review]
OK

**Pull request merged**